ansible/roles_ocp_workloads/ocp4_workload_codeserver/tasks/workload.yml

```diff
@@ -24,12 +24,10 @@
 - name: Set variables
   ansible.builtin.set_fact:
     _ocp4_workload_codeserver_wildcard_domain: "{{ r_ingress_controller.resources[0].status.domain }}"
-    _ocp4_workload_codeserver_gitea_url: >-
-      {{ r_gitea.resources[0].status.giteaRoute }}
     _ocp4_workload_codeserver_gitea_hostname: >-
       {{ r_gitea.resources[0].status.giteaHostname }}
     _ocp4_workload_codeserver_gitea_protocol: >-
-      {{ 'https' if r_gitea.resources[0].status.giteaRoute is search('https') else 'http' }}
+      {{ 'https' if r_gitea.resources[0].status.giteaHostname is search('https') else 'http' }}
```

Comment:
This won't work. It's still `status.giteaRoute` for the full URL including http(s)